viz.yaml

```diff
@@ -296,12 +296,21 @@ publish:
     depends: "lib-header-css"
     context:
       title: "Water use in the U.S."
+  -
+    id: static_story_figure
+    location: "images/WU_stories_static.png"
+    mimetype: image/png
+    export: true
+    title: "water use in the U.S."
+    alttext: "water use in the U.S."
+    relpath: "images/WU_stories_static.png"
   -
     id: key_stories
     template: layout/templates/key_stories.mustache
-    depends: ["rank_states_utils", "national_pie", "wu_data_15_sum_json", "styles"]
+    depends: ["rank_states_utils", "national_pie", "wu_data_15_sum_json", "styles", "static_story_figure"]
     context:
       sections: ["rank_states_utils", "national_pie"]
+      figure: "static_story_figure"
```

Comment:
Not sure why we'd veer too far from templating. The framework forces us to use alttext and title for publishing an image (or that is what I think is required, since that is the error I was getting w/o it), so we should use those required fields (use the templates) or not require the fields.

Reply:
I think templating is generally a good idea.

Reply:
I followed the example you shared here: https://github.com/USGS-VIZLAB/how-to-gdp/blob/master/layout/templates/gdp-figure.mustache
What do you want to call it?

Reply:
Right, that's because I reused that exact template over and over for multiple figures. You added 

```
<figure>
    {{{ figure }}}
</figure>
```

directly under the <div> for this figure, so you should probably be more specific about which figure is being passed in just in case we end up with more than one. So maybe change it to:

```
<figure>
    {{{ pie-map-fig }}}
</figure>
```

Reply:
updated